aws-datastore/src/test/java/com/amplifyframework/datastore/appsync/AppSyncMocking.java

```diff
@@ -101,7 +101,7 @@ public static CreateConfigurator onCreate(AppSync mock) {
 
                 // Pass back a ModelWithMetadata. Model is the one provided.
                 ModelMetadata metadata =
-                    new ModelMetadata(capturedModel.getId(), false, 1, Time.now());
+                    new ModelMetadata(capturedModel.getId(), false, 1, new Temporal.Timestamp());
```

Comment:
Why not your cool `Temporal.now()`, in all places, instead?

Reply:
Oops, missed this one!  Will change.